build.sh

```diff
@@ -18,12 +41,28 @@ zephyr_sdk_path="${this_dir}/zephyr-sdk-${zephyr_version}/"
 zephyr_build='macos-aarch64'
 zephyr_archive="zephyr-sdk-${zephyr_version}_${zephyr_build}.tar.xz"
 if [[ ! -d "$zephyr_sdk_path" ]]; then
+    header -B "Installing zephyr-sdk ==============================================="
+
     if [[ ! -f "$zephyr_archive" ]]; then
+        header -T -B "  Fetching zephyr-sdk"
         curl -LO "https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v${zephyr_version}/$zephyr_archive"
     fi
-    tar xf "$zephyr_archive"
+
+    header -T -B "  Untarring zephyr-sdk"
```

Comment:
Had never seen this term before - untarring

Reply:
Google says it's very common in this context.